daemon/container_operations_unix.go

```diff
@@ -19,13 +19,13 @@ import (
 	"github.com/docker/docker/pkg/mount"
 	"github.com/docker/docker/pkg/stringid"
 	"github.com/docker/docker/runconfig"
+	"github.com/docker/docker/vendor/src/github.com/opencontainers/specs/specs-go"
```

Comment:
This is a mis-rewrite of the imports via someone's editor plugin with a possibly wrong `GOPATH` setup. Line 28 has the correct import of `specs-go`.